src/Microsoft.ML.TensorFlow/doc.xml

```diff
@@ -0,0 +1,41 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<doc>
+  <members>
+
+    <member name="TensorflowTransform">
+      <summary>
+        Extracts hidden layers' values from a pre-trained Tensorflow model.
+      </summary>
+      <remarks>
+        The TensorflowTransform extracts the specified output from the operation computed on the graph (given the input(s)) using a pre-trained <a href="https://www.tensorflow.org">Tensorflow</a> model.
+        The transform takes as input the Tensorflow model together with the names of the inputs to the model and name of the operation for which output values will be extracted from the model.
+
+        The TensorflowTransform has following assumptions regarding the input, output and processing of data.
+        <list type="number">
+          <item>
+            The transform currently accepts the <a href="https://www.tensorflow.org/mobile/prepare_models">frozen TensorFlow model</a> file as input.
+          </item>
+          <item>The transform supports scoring only one example at a time.</item>
+          <item>The name of input column(s) should match the name of input(s) in Tensorflow model.</item>
```

Comment:
If the input columns have to match the names of the inputs in the TF model, is this parameter needed just to identify which subset of columns should be used? Would a user need to rename the columns before adding this transform?

Reply:
This is basically to map columns in IDataView to inputs in TF model. If the names are kept same we wont have to specify the mapping. The other possibility would be to have overloaded constructor where we can define which column in IDataView maps to which input to TF model using a dictionary.

Right now, yes column needs to be renamed before using this transform.

---
In reply to: [213780817](https://github.com/dotnet/machinelearning/pull/704#discussion_r213780817) [](ancestors = 213780817)

Reply:
I have created issue #769 to track this. For now, what Zeeshan said is correct. Perhaps we should add more details here, explaining how to rename data view columns.

---
In reply to: [213797869](https://github.com/dotnet/machinelearning/pull/704#discussion_r213797869) [](ancestors = 213797869,213780817)